blockbase/assets/alignments-front.css

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This file would hopefully be removed once Gutenberg provides front-end alignments. https://github.com/WordPress/gutenberg/issues/26633